### ISSUE TITLE
Season posters in view id 511 Circle InfoPanel

### ISSUE THREAD

**Issue by meesterexx** [marduklev/skin.swan-alpha#14]
Loving the skin and working on my build with it. Would it be possible to get large season posters to show in view 511 like they do in view 50? Currently it seems to only display show poster. View 510 shows large show poster and small season posters, but I think 511 would benefit from large season poster since the smaller circles aren't able to show the full art for the season. Also the ratings in 510 and 511 don't match, 510 shows 4 and 511 only shows 1.

And / or alternately could we get the option to show poster art in 50 on the right so it better matches infopanel?

Here is how it dispalays the same show seasons in TMDBHelper (and other addons):
View 511 season:
![screenshot00000](https://user-images.githubusercontent.com/13154733/156631678-18e54d31-f922-413e-91de-59a35a138514.png)

View 510 Season:
![screenshot00002](https://user-images.githubusercontent.com/13154733/156632151-48bcf737-0155-43f9-866a-8009e4d9afa3.png)

View 50 Season:
![screenshot00001](https://user-images.githubusercontent.com/13154733/156631713-baa88359-77a1-4077-a61d-89764b7557f5.png)

**Comment by marduklev** [comment]
Hi, glad you loved it. It'll get better from time to time. ;-)

I did small change, which should at least gives more diversity. 
![img](https://i.imgur.com/GVDaHbM.png)

https://github.com/marduklev/skin.swan-alpha/commit/307a462a927772fa6a5a02bc3cedb49330c2aa83

The 'issue' is more that this viewtypes are use fanart or keyart (poster with no name ) instead of posters for a 'unique' combination of art n labels ( matter of taste tough ;-) ).
So when scanned to lib and dedicated artwork exist it'll should look like ..
![img](https://i.imgur.com/obAy35T.png)


So for the ratings, i got no isses so far on my end, refreshing wise ( tough some changes in var need be done later on and i'll take a look whats causing it, as i am wondered about te difference ( visible on view 50, hidden in view 51** ))

- Did you check that you enabled the fallback using 'themoviedb.helper' in the skinsettings ? )

**Comment by meesterexx** [comment]
Looks amazing, thanks! I'm not having the ratings issue now. I did have tmdbhelper fallback already set, maybe it was  a cache issue, I did clear cache and thumbnails after updating to latest github commit and issue did not recur.

So for my TV Show/Season/Episodes I use views 511/511/56 combined with circle widgets on the spotlight home. Makes for a deisgn that I feel really pops and is both beautiful and distinct from other skins out there. So thank you for all your work!

Here's how I implemented it:
![screenshot00000](https://user-images.githubusercontent.com/13154733/157060000-5a525d93-dd61-49b8-afb0-e643fa3b3e7a.png)

![screenshot00003](https://user-images.githubusercontent.com/13154733/157060014-2954a56a-6e4c-4b61-9a00-40e533ea4466.png)

![screenshot00001](https://user-images.githubusercontent.com/13154733/157060037-5f63839b-8e91-44f9-8067-b74a3230308b.png)

![screenshot00002](https://user-images.githubusercontent.com/13154733/157060069-0eb3a007-ba1c-4dce-b30d-774669651d09.png)
